lib/packages/core/src/api/factories/transaction/sendAmountToMultipleRecipients.ts

```diff
@@ -0,0 +1,58 @@
+/** @module core.api.factories */
+
+/**
+ * Copyright (c) 2019 Burst Apps Team
+ */
+import {BurstService} from '../../../service/burstService';
+import {TransactionId} from '../../../typings/transactionId';
+import {TransactionResponse} from '../../../typings/transactionResponse';
+import {MultioutRecipientAmount} from '../../../typings/multioutRecipientAmount';
+import {signAndBroadcastTransaction} from '../../../internal/signAndBroadcastTransaction';
+import {DefaultDeadline} from '../../../constants';
+
+function mountRecipientsString(recipientAmounts: MultioutRecipientAmount[]): string {
```

Comment:
better mount the string internally, and not delegate implementation details to the user.